### PR TITLE
Add Perl-5.32.1-GCCcore-10.3.0.eb to NESSI/2022.11

### PR DESCRIPTION
**Pull request by TopRichard**
Trying to add Perl-5.32.1-GCCcore-10.3.0.eb to NESSI/2022.11

### CODE REVIEW

File: EESSI-pilot-install-software.sh [hunk 1]
```diff
@@ -424,7 +424,7 @@ echo_green "All set, let's start installing some software in ${EASYBUILD_INSTALL
 #$EB Perl-5.32.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2640
 #use enhanced CMake easyblock to patch CMake's UnixPaths.cmake script if --sysroot is set
 #from https://github.com/easybuilders/easybuild-easyblocks/pull/2248
-$EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
+#$EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
```
Comment (trz42) [inline EESSI-pilot-install-software.sh:427]: Was there a specific reason to comment out building CMake?

File: EESSI-pilot-install-software.sh [hunk 2]
```diff
@@ -449,6 +449,7 @@ $EB CMake-3.20.1-GCCcore-10.3.0.eb --robot --include-easyblocks-from-pr 2248
 #####################
 ### add packages here
 #####################
+$EB Perl-5.32.1-GCCcore-10.3.0.eb --robot
```
Comment (trz42) [inline EESSI-pilot-install-software.sh:452]: There is a commented out line for Perl in the script. Should we use that one? Also includes some additional parameters.

File: EESSI-pilot-install-software.sh (hunk 1 above)
Comment (trz42) [inline EESSI-pilot-install-software.sh:427]: Was there a specific reason to comment out building CMake?

File: eessi-2022.11.yml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 easyconfigs:
   - EasyBuild-4.7.0.eb
-  - CMake-3.20.1-GCCcore-10.3.0.eb
```
Comment (trz42) [inline eessi-2022.11.yml:3]: Any specific reason to remove CMake from the easy stack file?

Also, we might have forgotten to add the parameter `--include-easyblocks-from-pr 2248` here (in the PR that got merged, my fault for not spotting it earlier). We should make a small PR before merging the others to get this fixed.

**Review by trz42 — CHANGES_REQUESTED** [review]
Left a few comments. Maybe just clarifications are needed, maybe changes are needed. Will see.

File: EESSI-pilot-install-software.sh (hunk 1 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh:427]: It is already built and present in the CernVM-FS.

File: EESSI-pilot-install-software.sh (hunk 2 above)
Comment (TopRichard) [reply EESSI-pilot-install-software.sh:452]: Yep, I will create a new pull request using the one within the script.

File: EESSI-pilot-install-software.sh (hunk 1 above)
Comment (trz42) [reply EESSI-pilot-install-software.sh:427]: Hmm, that's not a reason to comment it out. If it is present, we *can* comment it out, but we should only do so if there is a good reason (such as hitting the rate limit issue).

Can you change it and rebuild?

Comment (TopRichard) [reply EESSI-pilot-install-software.sh:427]: sure

File: eessi-2022.11.yml (hunk 3 above)
Comment (TopRichard) [reply eessi-2022.11.yml:3]: Followed the new guidelines and added it back

**Comment by trz42** [comment]
The Fram jobs have been canceled. So need to rebuild them. Unsetting/setting `bot:build` label.

**Comment by trz42** [comment]
For the time being we ignore the build jobs on `AWS-NESSI`.

@TopRichard Some of the tarballs have a reasonable size, some seem to not contain anything/much. Could you investigate the reason(s)?

**Comment by trz42** [comment]
The software was a dependency of #20 which has already been ingested. Hence, we close this here.

**Pull request closed without merging**